### ISSUE TITLE
Build error on Linux with .NET Core SDK 3

### ISSUE THREAD

**Issue by mfkl** [novotnyllc/MSBuildSdkExtras#207]
Hi,

Minimal repro here: https://github.com/mfkl/linux-sdk-extras-netcore3

```
Microsoft (R) Build Engine version 16.4.0+e901037fe for .NET Core
/usr/share/dotnet/sdk/3.1.102/Sdks/Microsoft.NET.Sdk.WindowsDesktop/targets/Microsoft.NET.Sdk.WindowsDesktop.props(208,3): error MSB4019: The imported project "/usr/share/dotnet/sdk/3.1.102/Sdks/Microsoft.NET.Sdk.WindowsDesktop/targets/Microsoft.WinFX.props" was not found. Confirm that the expression in the Import declaration "Microsoft.WinFX.props" is correct, and that the file exists on disk. 
    0 Warning(s)
    1 Error(s)
```

dotnet version: 3.1.102
ubuntu version: 18.04.2 LTS

Works fine with `<Project Sdk="Microsoft.NET.Sdk">` but I need to use MSBuildSdkExtras for multi targeting.
Works fine with dotnet 2.1.700.

Probably related https://github.com/novotnyllc/MSBuildSdkExtras/pull/133

**Comment by mfkl** [comment]
dup https://github.com/dotnet/sdk/issues/11108

**Comment by Nirmal4G** [comment]
The Fix is here: dotnet/wpf#2975
